cscs-checks/libraries/magma/magma_checks.py

```diff
@@ -41,79 +41,67 @@ def __init__(self, subtest):
             }
         elif subtest == 'zgemm':
             self.perf_patterns = {
-                'magma': sn.extractsingle(r'MAGMA GFlops: (?P<magma_gflops>\S+)',
-                                          self.stdout, 'magma_gflops', float),
+                'magma': sn.extractsingle(
+                             r'MAGMA GFlops: (?P<magma_gflops>\S+)',
+                             self.stdout, 'magma_gflops', float, 2
+                         ),
                 'cublas': sn.extractsingle(
                     r'cuBLAS GFlops: (?P<cublas_gflops>\S+)', self.stdout,
-                    'cublas_gflops', float),
-                'cpu': sn.extractsingle(r'CPU GFlops: (?P<cpu_gflops>\S+)',
-                                        self.stdout, 'cpu_gflops', float)
+                    'cublas_gflops', float, 2)
             }
             self.reference = {
                 'daint:gpu': {
-                    'magma':  (3344, -0.05, None, 'Gflop/s'),
-                    'cublas': (3709, -0.05, None, 'Gflop/s'),
-                    'cpu':    (42.8, -0.27, None, 'Gflop/s'),
+                    'magma':  (3749, -0.05, None, 'Gflop/s'),
+                    'cublas': (4050, -0.05, None, 'Gflop/s'),
```

Comment:
Are you sure about your reference values? I have updated them in Apr. using some statistical data.

Reply:
The old values are outdated unfortunately, since the case is executed with 3 different matrix sizes, and the 3rd is used. In the past it was the 1st. The alternative would be to let the loop count down:
for( int itest = 3-1; itest >= 0; itest-- )

Reply:
That's fine. In any case we would have to update them.